pkg/test/matchers/golden/update_files.go

```diff
@@ -16,5 +16,5 @@ func RerunMsg(path string) string {
 	if err != nil {
 		absPath = path + " Failed to retrieve cwd"
 	}
-	return fmt.Sprintf("Rerun the test with UPDATE_GOLDEN_FILES=true flag to update file: %s. Example: make test UPDATE_GOLDEN_FILES=true", absPath)
+	return fmt.Sprintf("Rerun the test with UPDATE_GOLDEN_FILES=true flag to update file: %s. Example: make test UPDATE_GOLDEN_FILES=true or make test/transparentproxy UPDATE_GOLDEN_FILES=true", absPath)
```

Comment:
I think we should add new entry in https://github.com/kumahq/kuma/blob/738b743bc365f49d9af6cdbc012ffb4b76dd1ba1/.github/workflows/pr-comments.yaml#L63 to utilize github action for this

Reply:
Addressed